### PR TITLE
Update Monetization Subsection

### PR DESCRIPTION
**Pull request by Yung-Beef**

### CODE REVIEW

**Comment by Yung-Beef** [comment]
Monetization sub-section of Architecture

File: docs/lightpaper/architecture/monetization/Tips.md [hunk 1]
```diff
@@ -4,9 +4,12 @@ title: Tips
...
 
 Similar to tips on YouTube's Superchat – it is possible to use tips to highlight users’ comments.
-Highlighted comments stand out compared to the free comments such that content creators will
+Highlighted comments stand out compared to free comments such that content creators will
 notice those comments first. If the creators have thousands of comments per post as we see
-with people like PewDiePie, highlighting will make a user’s comment stand out.
\ No newline at end of file
+with people like PewDiePie, highlighting will make a user’s comment stand out.
+
+Tips will go straight to the account that you are tipping, and the tippee will receive the full value of the tip, as there are no middlemen to take a cut.
```
Comment (F3Joule) [inline docs/lightpaper/architecture/monetization/Tips.md:14]: As stated, there will be no cut from the tip directly, but the sender will pay a fee for the blockchain operation, so in a result it sounds not fully clear.

**Review by F3Joule — APPROVED** [review]

File: docs/lightpaper/architecture/monetization/Tips.md (hunk 1 above)
Comment (Yung-Beef) [reply docs/lightpaper/architecture/monetization/Tips.md:14]: Yeah but if I tip 1 KSM they get 1 KSM, I just pay 1.0001 KSM, so they still get the full value.

File: docs/lightpaper/architecture/monetization/SocialTokens.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 Tokenize your Subsocial space with social tokens in order to help build and monetize your
 space. Your followers can co-own your space and, if desired, participate in governance. A
 co-owned space is essentially a DAO. This is something we will touch upon in a later section.
 
-A built-in AMM DEX for social tokens, that’s native to Subsocial, will let users easily exchange
-social tokens without having to worry about trying to trade their own tokens from one person to
-another.
+Tokens will either be minted from a smart contract using a bonding curve, or will be created by the owner of the space and listed for sale on a DEX.
+We are still determining how this will work, but if the later option is chosen, we will either build our own DEX that is native to Subsocial, 
+or utilize Kusama's cross-chain messaging protocol to move those tokens to another chain that has a DEX.
```
[the remark this answers is not in the document]
Comment (Yung-Beef) [reply docs/lightpaper/architecture/monetization/SocialTokens.md]: Updated: https://github.com/dappforce/subsocial-docs/pull/26/commits/a70769ee68fa5da5116ca62720973850b9de3cc2

I think it committed to this PR right, please double check, Github still confuses me...

File: docs/lightpaper/architecture/monetization/SmartContracts.md [hunk 3]
```diff
@@ -10,11 +10,14 @@ It is possible to create different monetization algorithms that are more advance
...
 most popular posts based on the number of likes and shares.
 
 We hope to capture the same level of creativity as seen in the DeFi space on Ethereum. We
 recognise that without smart contracts Ethereum DeFi would not be possible. The interaction
-between many DeFi projects is an expression of the composability of smart contracts.
\ No newline at end of file
+between many DeFi projects is an expression of the composability of smart contracts.
+
+We will either integrate smart contract functionality into the Subsocial chain, 
+or integrate with other parachains that support smart contracts, like Moonriver.
```
Comment (siman) [inline docs/lightpaper/architecture/monetization/SmartContracts.md:23]: We are currently looking into ways to potentially integrate smart contract functionality into Subsocial.

**Review by siman — APPROVED** [review]

File: docs/lightpaper/architecture/monetization/SmartContracts.md (hunk 3 above)
Comment (siman) [inline docs/lightpaper/architecture/monetization/SmartContracts.md:23]:
```suggestion
We are currently looking into ways to potentially integrate smart contract functionality into Subsocial.
```

**Review by siman — CHANGES_REQUESTED** [review]

**Pull request merged**